JavaFiddle-ejb/src/main/java/ru/javafiddle/core/ejb/GroupBean.java

```diff
@@ -146,31 +136,160 @@ private int getUserId(String userNickName) {
 
     private User getUser(String userNickName) {
```

Comment:
Avoid such stuff. User entity should come as an external param into your Group bean.
And fetching it from DB is the UserBean's job.